src/impurity_init.ipp

```diff
@@ -362,7 +362,7 @@ void HybridizationSimulation<IMP_MODEL>::read_eq_time_two_particle_greens_meas()
     return;
   }
 
-  std::ifstream infile_f(boost::lexical_cast<std::string>(par[fname_key]).c_str());
+  std::ifstream infile_f(par[fname_key].as<std::string>());
```

Comment:
Apparently i was wrong in my previous comments, and at least clang does require the `template` keyword here (see https://travis-ci.org/ALPSCore/CT-HYB/jobs/355893268#L5641) --- my apologies. Oh well, we'll fix this...